Aquality.Selenium.Core/src/Aquality.Selenium.Core/Visualization/DumpManager.cs

```diff
@@ -93,7 +95,7 @@ public virtual void Save(string dumpName = null)
                 {
                     try
                     {
-                        element.Value.Visual.Image.Save(Path.Combine(directory.FullName, $"{element.Key}.png"));
+                        element.Value.Visual.Image.Save(Path.Combine(directory.FullName, $"{element.Key}{ImageFormat}"));
```

Comment:
Don't we need to specify the ImageFormat parameter, so that image would be save in a proper format?
Like it is described here: http://csharphelper.com/blog/2014/07/save-images-with-an-appropriate-format-depending-on-the-file-names-extension-in-c/